docs/guide/coordinates/index.rst

```diff
@@ -24,183 +24,6 @@ See the following pages for detailed discussion of specific topics:
    velocities
    wcs
 
-Getting Started
-===============
-
-The easiest interface to work with coordinates is through the `~astropy.coordinates.SkyCoord` class::
-
-  >>> import astropy.units as u
-  >>> from astropy.coordinates import SkyCoord
-  >>> from sunpy.coordinates import frames
-
-  >>> c = SkyCoord(-100*u.arcsec, 500*u.arcsec, frame=frames.Helioprojective)
-  >>> c = SkyCoord(x=-72241.0*u.km, y=361206.1*u.km, z=589951.4*u.km, frame=frames.Heliocentric)
-  >>> c = SkyCoord(70*u.deg, -30*u.deg, frame=frames.HeliographicStonyhurst)
-  >>> c
-  <SkyCoord (HeliographicStonyhurst: obstime=None, rsun=695700.0 km): (lon, lat) in deg
-      (70., -30.)>
-
-
-It is also possible to use strings to specify the frame but in that case make sure to
-explicitly import `sunpy.coordinates` as it registers SunPy's coordinate frames with
-the Astropy coordinates framework::
-
-  >>> import astropy.units as u
-  >>> from astropy.coordinates import SkyCoord
-
-  >>> import sunpy.coordinates
-  >>> c = SkyCoord(-100*u.arcsec, 500*u.arcsec, frame='helioprojective', observer='earth')
-  >>> c
-  <SkyCoord (Helioprojective: obstime=None, rsun=695700.0 km, observer=earth): (Tx, Ty) in arcsec
-      (-100., 500.)>
-
-
-`~astropy.coordinates.SkyCoord` and all coordinate frames
-support array coordinates. These work the same as single-value coordinates,
-but they store multiple coordinates in a single object. When you're going to
-apply the same operation to many different coordinates, this is a better choice
-than a list of `~astropy.coordinates.SkyCoord` objects, because it will be
-*much* faster than applying the operation to each
-`~astropy.coordinates.SkyCoord` in a ``for`` loop::
-
-   >>> c = SkyCoord([-500, 400]*u.arcsec, [100, 200]*u.arcsec, frame=frames.Helioprojective)
-   >>> c
-   <SkyCoord (Helioprojective: obstime=None, rsun=695700.0 km, observer=None): (Tx, Ty) in arcsec
-       [(-500.,  100.), ( 400.,  200.)]>
-   >>> c[0]
-   <SkyCoord (Helioprojective: obstime=None, rsun=695700.0 km, observer=None): (Tx, Ty) in arcsec
-       (-500.,  100.)>
```

Comment:
This should become a "how-to" guide as well: "How to create a Coordinate object". I would argue this isn't (or shouldn't be) covered in the tutorial because it shows the variety of ways to accomplish the same task.